zerver/models/scheduled_jobs.py

```diff
@@ -253,3 +253,18 @@ def to_dict(self) -> Union[APIScheduledStreamMessageDict, APIScheduledDirectMess
     "digest": ScheduledEmail.DIGEST,
     "invitation_reminder": ScheduledEmail.INVITATION_REMINDER,
 }
+
+
+class ScheduledUserStatus(models.Model):
+    user_profile = models.OneToOneField(UserProfile, on_delete=CASCADE)
+
+    scheduled_timestamp = models.DateTimeField(db_index=True)
+    client = models.ForeignKey(Client, on_delete=CASCADE)
```

Comment:
Why does this not subclass `AbstractScheduledJob`?

Reply:
well, I felt this is similar to `ScheduledMessages` -- a `realm` field won't be needed since you would be clearing out status of own realm users, and a data field - I don't think so it is needed. Should I update it?

Reply:
ok... updated!